pandas/core/frame.py

```diff
@@ -3035,6 +3035,129 @@ def reindex_axis(self, labels, axis=0, method=None, level=None, copy=True,
                                         method=method, level=level, copy=copy,
                                         limit=limit, fill_value=fill_value)
 
+    def drop(self, labels=None, axis=0, index=None, columns=None,
+             level=None, inplace=False, errors='raise'):
+        """
+        Drop rows or columns.
```

Comment:
Would like this to mention *how* the rows are determined. Maybe `Drop specified labels from rows or columns`.

Reply:
some leftovers from the other PR here. But you can give the comment on https://github.com/pandas-dev/pandas/pull/20120